### ISSUE TITLE
Hi, Can you work on CPU?

### ISSUE THREAD

**Issue by flystarhe** [matterport/Mask_RCNN#646]
Thanks

**Comment by ad12** [comment]
Yup, set the `GPU_COUNT=1` in your extension of the `Config `class

**Comment by zia-ghiasi** [comment]
What if I want to use 1 GPU?

**Comment by Houd1ny** [comment]
@zia-ghiasi You should set GPU_COUNT=2 as far I know

**Comment by dhanush-ai1990** [comment]
Its so weird that when i run inference on CPU, it doesnt not output all the bounding boxes.

For example on gpu inference all 3 bounding boxes are returned, with cpu only one. Why is this happening ?
